scripts/BuildDocs.ps1

```diff
@@ -11,18 +11,47 @@ $docsRepoPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\docs\_repo")
 # Ensure the tools path exists
 mkdir $toolsPath -Force | Out-Null
 
+if (![System.IO.File]::Exists($docfxBinPath)) {
+    mkdir $docfxBinPath -Force | Out-Null
+}
+
 if (![System.IO.File]::Exists($docfxExePath)) {
     # Download DocFX
-    Remove-Item -Path "$docfxBinPath" -Force -ErrorAction Stop | Out-Null
-    (new-object net.webclient).DownloadFile('https://github.com/dotnet/docfx/releases/download/v1.8/docfx.zip', "$docfxZipPath")
+    if (Test-Path -Path $docfxBinPath) { Remove-Item -Path "$docfxBinPath" -Force -ErrorAction Stop | Out-Null }
+    # Github uses TLS 1.2
+    $originalSecurityProtocol = [Net.ServicePointManager]::SecurityProtocol
+    try {
+        [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12
+        Write-Verbose "Downloading docfx..."
+        # The docFX 1.x release does generate documentation but is incomplete as it doesn't understand dotNet 2.1
+        # The docFX 2.x release errors during generation as it doesn't understand dotNet 2.1
+        # 2.1 support seems to be slated for docfx 3.x release - https://github.com/dotnet/docfx/projects/1
+        #
+        # For now use the 1.x series of docfx
+        (new-object net.webclient).DownloadFile('https://github.com/dotnet/docfx/releases/download/v1.9.4/docfx.zip', "$docfxZipPath")
```

Comment:
No idea.  This is just a whitespace change of the original code from @daviwil 3 years ago. https://github.com/PowerShell/PowerShellEditorServices/blame/f847e2fd64f89edcc089803dd03775409e511344/scripts/BuildDocs.ps1#L17

Reply:
Ok, up to you, but `Invoke-WebRequest` would be my preference here

Reply:
Given the age, it was probably due to IWR being SOO slow compared to WebClient

Reply:
I think there were (are) issues with IWR in the Windows PowerShell and early versions of PowerShell Core.  Do we still need to build on a system running PS 5.1?

Reply:
Fixed.  Changed to IWR.

Reply:
Ah didn't know about issues. Early versions of PSCore won't be a problem since they're out of support in a few months. WinPS I have no idea, but performance shouldn't be an issue for official builds at least.

Reply:
I tested on WinPS (I haven't made the change PSCore yet ⏲).